.prettierrc.json

```diff
@@ -1 +1,5 @@
-{}
+{
+  "tabWidth": 2,
+  "useTabs": false,
+  "trailingComma": "es5"
```

Comment:
I remember I briefly looked into this issue several weeks ago and found in the docs for [the trailingComma option](https://prettier.io/docs/en/options.html#trailing-commas): "Default value changed from es5 to all in v3.0.0," but I didn't figure out why the version of prettier that we were running was inconsistent. 👍

Reply:
yeah, what a weird bug. dep of a dep causing conflicts. just goes to show: Don't fully trust migration tools! I didn't expect prettier would be stealth-upgraded by another library. Will have to check eg `package-lock.json` more thoroughly next time. And will also have to study further how diff versions ran in diff envs at some point (maybe `prettier-fallback` from Storybook, unsure). 

also, it may make sense to define all of our prettier config explicitly instead of relying on defaults, although that will only help with breaking changes that alter config defaults.